tests/test_params_object.py

```diff
@@ -1,6 +1,11 @@
+import sys
+sys.stdin = open('/dev/tty')
+if 'safely_exp.utils.auth_utils' not in sys.path:
```

Comment:
Please see  [link](https://github.com/pulkitag/exp_params_mng/blob/master/README.md#installation) Put these two lines in `db_config.py`.